_release/bat/storage_bat/ceph_test.go

```diff
@@ -105,3 +116,23 @@ func TestCopyBlockDevice(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestMain(m *testing.M) {
+	data, err := ioutil.ReadFile("/etc/ciao/configuration.yaml")
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error reading config file: %v", err)
+		os.Exit(1)
+	}
```

Comment:
What happens if we exit here?  Are all the tests in this package reported as failed?  If so it might be better not to exit but just not to initialise driver.ID or rather leave it set to "".  This way each test could check to see if driver.ID == "" and if it is called t.Skip().  This should give us a clean run when BAT is not being run on the scheduler node.